core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -247,13 +260,32 @@ class CityInfoReligionManager {
             cityInfo.civInfo.gameInfo.getCities()
                 .filter {
                     it != cityInfo
-                            && it.getCenterTile().aerialDistanceTo(cityInfo.getCenterTile()) <= 10
+                    && it.getCenterTile().aerialDistanceTo(cityInfo.getCenterTile()) <= it.religion.getSpreadRange()
                 }
         for (city in allCitiesWithin10Tiles) {
             val majorityReligionOfCity = city.religion.getMajorityReligion() ?: continue
             if (!majorityReligionOfCity.isMajorReligion()) continue
-            addedPressure.add(majorityReligionOfCity.name, pressureFromAdjacentCities)
+            addedPressure.add(majorityReligionOfCity.name, city.religion.pressureAmountToAdjacentCities(cityInfo))
         }
         return addedPressure
     }
+    
+    private fun pressureAmountToAdjacentCities(pressuredCity: CityInfo): Int {
+        var pressure = pressureFromAdjacentCities.toFloat()
+        
+        for (unique in cityInfo.getMatchingUniques("[]% Natural religion spread to []")) {
+            if (pressuredCity.matchesFilter(unique.params[1]))
+                pressure *= 1f + unique.params[0].toFloat() / 100f
+        }
+        
+        for (unique in cityInfo.getMatchingUniques("[]% Natural religion spread to [] after discovering []"))
```

Comment:
If this accepts policies as well (which is a good idea) then maybe change to "with x"